### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by tessi**
prepares the release of v1.3.0.

I will tag and publish 1.3.0 once this was merged.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 /pkg/
 /spec/reports/
 /tmp/
+/*.gem
```
Comment (tessi) [inline .gitignore:10]: to not accidentally push the built gem

File: rubocop-bitcrowd.gemspec [hunk 2]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.bindir        = 'exe'
   spec.executables   = 'rubocop-autofix'
 
-  spec.add_runtime_dependency 'rubocop', '>= 0.56'
+  spec.add_runtime_dependency 'rubocop', '~> 0.56'
```
Comment (tessi) [inline rubocop-bitcrowd.gemspec:19]: using `>=` results in the following warning when building the gem:

```
> gem build rubocop-bitcrowd.gemspec
WARNING:  open-ended dependency on rubocop (>= 0.56) is not recommended
  if rubocop is semantically versioned, use:
    add_runtime_dependency 'rubocop', '~> 0.56'
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
```

Comment (klappradla) [reply rubocop-bitcrowd.gemspec:19]: Ok, just learned, the thing is called
> twiddle-wakka

Makes perfect sense 🖤

**Review by klappradla — APPROVED** [review]
⛵️💨

**Pull request merged**